ts/ts/data.ts

```diff
@@ -1 +1,31 @@
 /* exported data */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+interface SavedHoliday {
+  name: string;
+  desc: string;
+  favorite: boolean;
+  imageReference: string;
+}
+
+interface Data {
+  holidays: SavedHoliday[];
+}
+
+const data: Data = {
+  holidays: [],
+};
+
```

Comment:
Since you're not using the retrieved data yet it's fine, but you can just assign the return of calling your `retrieveData` function directly to your `data` variable. Your `retrieveData` function either returns the data with values, or it returns the object with an empty array in your holidays property.